### PR TITLE
[CYP] Adds Cypress & configures Travis runs post merge

### PR DESCRIPTION
**Pull request by s-henney**
- Adds tests to confirm the expected pre-search content block is displayed
- Adds configuration to run the cypress tests through Travis.ci after a PR in this repo is merged
- Currently only configured to run tests against int. Once we have Netlify builds for other environments, we'll enable runs against those environments too.

### CODE REVIEW

File: cypress/Contentful/Fields/TextField.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return this.text.localeCompare(textField.text);
+  }
+
+  get text() {
+    return this.hasContent ? this._content : '';
+  }
+
+  get normalized() {
+    if (this._normalized === undefined) {
+      this._normalized = Formatter.normalizeText(this._content);
```
Comment (lukemoderwell) [inline cypress/Contentful/Fields/TextField.js]: What does normalizing the text do?

File: cypress/integration/pre_search_content_spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { ContentfulElementValidator as Element } from '../Contentful/ContentfulElementValidator'
+import { ContentBlockManager } from '../Contentful/Models/ContentBlockModel';
+
+function preSearchContentBlockShouldBeDisplayed(contentBlock) {
+  cy.get('app-suggested').find('.suggested-container').as('preSearchContentBlock');
+  Element.shouldContainText('preSearchContentBlock', contentBlock.content);
```
Comment (lukemoderwell) [inline cypress/integration/pre_search_content_spec.js]: Does `shouldContainText` also check HTML/markdown? I think this response might be MD but the `.suggested-container` will be HTML.

File: cypress/Contentful/Fields/TextField.js (hunk 1 above)
Comment (s-henney) [reply cypress/Contentful/Fields/TextField.js]: It strips the markdown/html/etc. out and just gives you the unformatted human text. If we didn't do this, comparing Contentful data (which may contain formatting characters or just plain text) with the text we retrieved from the page's element (which may contain other formatting characters) would almost never match.

File: cypress/integration/pre_search_content_spec.js (hunk 2 above)
Comment (s-henney) [reply cypress/integration/pre_search_content_spec.js]: Behind the scenes Formatter.normalizeText() is run on both the content block text and the page element text before comparing, so in the end just the plain text for both is being compared.
normalizeText() uses the remove-markdown gem, fake renders the result in browser with "document.createElement('textarea');", and ends with some regex to strip out extra whitespace. This is the same code we're using in crds-net and it seems pretty stable.

**Pull request merged**